django_freeradius/apps.py

```diff
@@ -1,6 +1,17 @@
 from django.apps import AppConfig
+from django.core.exceptions import ImproperlyConfigured
+
+from .settings import API_TOKEN
 
 
 class DjangoFreeradiusConfig(AppConfig):
     name = 'django_freeradius'
     verbose_name = 'Freeradius'
+
+    def check_settings(self):
+        if API_TOKEN and len(API_TOKEN) < 15 or not API_TOKEN:    # pragma: nocover
+            raise ImproperlyConfigured('Security error: API_TOKEN is either not set or '
+                                       'is less than 15 characters.')
```

Comment:
the error message should mention the public name of the setting, `DJANGO_FREERADIUS_API_TOKEN`, the user won't be sure of what `API_TOKEN` is.